src/ERC20Rebasing.sol

```diff
@@ -116,7 +116,11 @@ abstract contract ERC20Rebasing is ERC20 {
         // Check overflow
         if (totalSharesAfter < totalSharesBefore) revert TotalSupplyOverflow();
         // Check total supply limit, can also revert with FullMulDivFailed in sharesToBalance
-        if (sharesToBalance(totalSharesAfter) > maxSupply()) revert TotalSupplyOverflow();
+        // Round up for total supply limit check
```

Comment:
Can we rephrase this a bit? `sharesToBalance` isn't used here anymore, and I think this would be confusing for a first time reader

Reply:
changed to reference fullMulDivUp